content/en/observability_pipelines/production_deployment_overview/_index.md

```diff
@@ -1,42 +1,188 @@
+
+#### When Observability Pipelines Worker should integrate with agents
+
+The Observability Pipelines Worker should integrate with agents that produce vendor-specific data that the Observability Pipelines Worker cannot replicate.
+
+For example, Datadog [Network Performance Monitoring][4] integrates the Datadog Agent with vendor-specific systems and produces vendor-specific data. Therefore, the Datadog Agent should collect the data and send it directly to Datadog, since the data is not a supported data type in the Observability Pipelines Worker.
+
+As another example, the Datadog Agent collects service metrics and enriches them with vendor-specific Datadog tags. In this case, the Datadog Agent should send the metrics directly to Datadog or route them through the Observability Pipelines Worker. The Observability Pipelines Worker should not replace the Datadog Agent because the data being produced is enriched in a vendor-specific way.
+
+If you integrate with an agent, configure the Observability Pipelines Worker to receive data directly from the agent over the local network, routing data through the Observability Pipelines Worker. Use source components such as the `datadog_agent` or `open_telemetry` to receive data from your agents.
+
+{{< img src="observability_pipelines/production_deployment_overview/from_other_agents.png" alt="A diagram showing a node containing multiple services, other agents, and the Observability Pipelines Worker, where the services and agents are sending data to the Worker and the Worker is sending data out" style="width:35%;" >}}
+
+Alternatively, you can deploy the Observability Pipelines Worker on separate nodes as an aggregator. See [Choosing where to process data](#choosing-where-to-process-data) for more details.
+
+##### Reducing agent risk
+
+When integrating with an agent, configure the agent to be a simple data forwarder and route supported data types through the Observability Pipelines Worker. This reduces the risk of data loss and service disruption by minimizing the agent's responsibilities.
+
+## Processing data
+
+If you want to design an efficient pipeline between your Observability Pipelines Worker's sources and sinks, it helps to understand which types of data to process and where to process it.
+
+### Choosing which data to process
+
+You can use Observability Pipelines Worker to process logs, metrics, and traces. However, real-time, vendor-specific data, such as continuous profiling data, is not interoperable and typically does not benefit from processing.
+
+### Choosing where to process data
+
+Observability Pipelines Worker can be deployed anywhere in your infrastructure. Deploy it directly on your node as an agent for local processing, or on separate nodes as an aggregator for remote processing. Where the processing happens depends largely on your use case and environment.
+
+#### Local processing
```

Comment:
Delete this section

Reply:
To clarify, are you saing we should delete the entire sections for "Choosing where to process data
" and "Local processing", or only the "Local processing section"?

Reply:
Perfect. Thanks!